Tasks/UniversalPackagesV0/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
Throughout: I think we want to prefer "Universal Packages" over "universal packages" or "Universal packages". 
@alexmullans @mitchdenny is that correct?